### DISCUSSION TITLE
蓝灯专业版变为免费版

### DISCUSSION THREAD

**Discussion by coeurdeciel** [getlantern/lantern-forum-cn#241] [General]
你好 蓝灯专业版突然变成了免费版  并且注册邮箱迟迟收不到恢复码 请问该如何解决？

**Comment by MaxMKMax** [comment]
@coeurdeciel 谢谢您的反馈. 您方便提供我您购买时用的邮箱吗? 也是账号邮箱.
我帮您和团队说明这个问题。我帮你紧急解决问题。我现在马上通知技术团队! 但需要您现在给我邮箱一下. 谢谢.

**Reply by coeurdeciel** [reply]
835511900@qq.com

**Reply by coeurdeciel** [reply]
@Mike777Khoo 
 你好 一天后邮箱依旧收不到验证码 账户也没有恢复 请问贵公司遇到这样的问题大概要多久能处理好呢？

**Reply by MaxMKMax** [reply]
@coeurdeciel 好的. 已经帮您提交上去了. 谢谢您支持蓝灯! 我们会尽快帮您处理. 请留意邮件. 谢谢!

**Reply by coeurdeciel** [reply]
你好 一周后问题依旧没有得到解决 贵公司打算用多久来处理这个问题呢？

**Reply by MaxMKMax** [reply]
@coeurdeciel 您好. 其实我上周有帮您提交上去了, 我现在帮您追一下. 看他们有什么解决方案. 等一等, 好了我回复您. 同时您也留意邮箱.

**Reply by coeurdeciel** [reply]
你好 半个月过去了 贵公司打算用多久来处理这个问题呢？

**Reply by MaxMKMax** [reply]
@coeurdeciel 您好. 客服那里还没有给我答复. 我再帮您追问一下. 不好意思啊.

**Reply by MaxMKMax** [reply]
@coeurdeciel 您好. 客服说已经帮您的账号回复状态了.

**Comment by MaxMKMax** [comment]
@coeurdeciel 您好. 客服回我了. 他说他有发邮件让您给他提交付款细节或单据. 您还没回复他呢.

**Reply by coeurdeciel** [reply]
![捕获](https://github.com/getlantern/lantern-forum-cn/assets/160586759/da092ba8-616a-4dc4-9b74-02b8a918af8b)

**Reply by coeurdeciel** [reply]
邮件似乎因为技术原因被退回 无法向贵公司的邮箱发送 顺便一提 目前蓝灯不能使用 7.83版本是完全无效的

**Reply by coeurdeciel** [reply]
@Mike777Khoo 你好 20天了 请问贵公司还要多久来解决这个问题呢？

**Reply by MaxMKMax** [reply]
@coeurdeciel 你好。我可以知道您的帐户电子邮件是什么吗？我将帮助您申请专业版延期。

**Reply by coeurdeciel** [reply]
@Mike777Khoo   835511900@qq.com   这个问题楼上就有 不过被折叠了

**Reply by MaxMKMax** [reply]
@coeurdeciel 您好. 我帮您拿到激活码了. 我刚发邮件给你了. 请在邮件收!
from: mike@getlantern.org

**Reply by MaxMKMax** [reply]
@coeurdeciel 7.8.4 最新版本明天晚上就会有了.

**Reply by coeurdeciel** [reply]
@Mike777Khoo  收到

**Reply by coeurdeciel** [reply]
@Mike777Khoo  你好 之前续费时购买了蓝灯专业版27个月的合约
之后出现BUG 账户注销 用激活码重新激活后 合约为24个月
可以联系贵公司 问问看这种情况有没有处理方案
实在不行 那也算了

**Reply by MaxMKMax** [reply]
@coeurdeciel 嗯嗯. 好的. 我帮您问了. 等待客服回复我. 谢谢您!

**Reply by MaxMKMax** [reply]
@coeurdeciel 您好. 客服说已经帮您的账号回复27个月的合约状态了.

**Reply by coeurdeciel** [reply]
好的 谢谢了

**Comment by MaxMKMax** [comment]
@coeurdeciel 要不麻烦您提交给我. 贴在这里就好. 因为系统里找不到您的付款资料. 所以需要您提供一下. 谢谢哦

**Reply by set-aua** [reply]
你好 mike lantern mac os版本  程序无法启动

**Reply by MaxMKMax** [reply]
@set-aua 
您好. 问题应该已经解决了. 请到我们页面下载新版本.
https://github.com/getlantern/lantern

**Comment by lshengjian** [comment]
我下载了最新的稳定版本，打开后专业版不见了。我的邮箱地址:lsj178@139.com

**Reply by MaxMKMax** [reply]
@lshengjian 您好. 请您提供您的支付单据.
